src/App.tsx

```diff
@@ -0,0 +1,39 @@
+import DigitButton from './components/DigitButton';
+import { operations, DIGITS, OPERATIONS_LIST } from './constants';
+import { useCalculate } from './hooks';
+
+function App() {
+  const { total, handleClearClick, handleDigitClick, handleOperationClick } =
+    useCalculate();
+
+  return (
+    <div className="calculator">
+      <h1 id="total">
+        {Math.abs(total || 0) === Infinity ? '오류' : total || 0}
+      </h1>
+      <div className="digits flex">
+        {DIGITS.map((digit) => (
+          <DigitButton key={digit} digit={digit} onClick={handleDigitClick} />
+        ))}
+      </div>
+      <div className="modifiers subgrid">
+        <button className="modifier" onClick={handleClearClick}>
+          AC
+        </button>
+      </div>
+      <div className="operations subgrid">
+        {OPERATIONS_LIST.map((operation) => (
+          <button
+            key={operation}
+            className="operation"
+            onClick={() => handleOperationClick(operation)}
+          >
+            {operations[operation]}
+          </button>
```

Comment:
```suggestion
          <OperatorButton
            key={operator}
            operation={operator}
            onClick={handleOperatorClick}
          />
```
위 숫자에 대한 버튼을 분리해주신 것 처럼 연산자 버튼도 동일한 방식으로 분리해봐도 좋을 것 같아요~

Reply:
이 부분은 https://github.com/next-step/react-calculator/pull/4#discussion_r1101022248 이 코멘트에 대한 생각 정리후에 다시 분리해보겠습니다